src/Command/SmokerCommand.php

```diff
@@ -50,9 +50,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $pool = $builder->build($testMethods);
 
+        $allowSelfSignedCertificate = (bool) $input->getOption('allow-self-signed-certificate');
+
         // Run the list of tests
-        Loop::run(function () use ($runner, $pool) {
-            $runner->loop($pool);
+        Loop::run(function () use ($runner, $pool, $allowSelfSignedCertificate) {
```

Comment:
I would pass this option to the runner constructor instead (as done with the concurrency)